leash.go

```diff
@@ -133,9 +133,17 @@ func run(options GlobalOptions) {
 
 		realToBeSent := make(chan event.Event, 10*options.NumSenders)
 		go func() {
-			for ev := range modifiedToBeSent {
-				realToBeSent <- ev
+			wg := sync.WaitGroup{}
+			for i := uint(0); i < options.NumSenders; i++ {
+				wg.Add(1)
+				go func() {
+					for ev := range modifiedToBeSent {
+						realToBeSent <- ev
+					}
+					wg.Done()
+				}()
 			}
+			wg.Wait()
```

Comment:
See comment below in `dropEventField`, first.

I don't think this is what needs to happen, and I don't think we need `toBeSent`, `modifiedToBeSent`, *and* `realToBeSent`. I think we need one of them.

Pass `options` down into `sendToLibhoney`, and simplify `modifyEventContents` to take an `event.Event` instead of a `chan event.Event` and just... modify it, no return value. no channels, no waitgroups. Everything you're doing to the event is simple transformation, there's no reason for the *up to N* channels involved in things. And `modifyEventContents` can just be called from inside the `sendToLibhoney` loop when pulling off `toBeSent`.

**Simplify**. `honeytail` does lots of things but in the end it's a log parser/processor/transmitter, and most of it should be able to be expressed fairly straightforwardly.